docs/recipes/babelrc.md

```diff
@@ -30,6 +30,21 @@ You can override the default Babel configuration AVA uses for test transpilation
 }
 ```
 
+## Provide ES2015 Polyfills to your tests
+
+You might run into issues when relying on ES2015-features your current environment do not support. Since Ava isn't providing any polyfills per default you might want to enable them by adding `babel-polyfill` manually to the `require` section:
```

Comment:
This is a good start. I think it could be clearer on what AVA does out of the box, and what kind of features end up being missing. There's also a big downside to using `babel-polyfill`, which is that it impacts how your program behaves (not just the tests!)

What do you think about this:

> AVA lets you write tests using new JavaScript syntax, even on Node.js version that do not support it otherwise. However it doesn't add or modify built-ins. For example `Array.prototype.includes()` is not available in Node.js 4 even when you use AVA.
>
> You can opt in to these features by loading [Babel's Polyfill](https://babeljs.io/docs/usage/polyfill/) module. Note that this modifies the environment, which impacts how your program itself behaves.
>
> You can load `babel-polyfill` by adding it to AVA's `require` options: